projects/1.19/assets/sweetmagic/sweetmagic/lang/zh_cn.json

```
@@ -0,0 +1,1097 @@
+	"block.sweetmagic.magiawood_slab": "魔法木板台阶",
+	"block.sweetmagic.magiawood_stairs": "魔法木板楼梯",
+	"block.sweetmagic.twilight_alstroemeria": "暮光六出",
+	"block.sweetmagic.sugarbell_plant": "糖铃植株",
+	"block.sweetmagic.sannyflower_plant": "阳光花植株",
+	"block.sweetmagic.moonblossom_plant": "月光植株",
+	"block.sweetmagic.drizzly_mysotis_plant": "细雨勿忘我植株",
+	"block.sweetmagic.fire_nasturtium_plant": "火金莲植株",
+	"block.sweetmagic.clerodendrum_plant": "紫蝶植株",
+	"block.sweetmagic.cotton_plant": "棉花植株",
+	"block.sweetmagic.glowflower_plant": "荧光花植株",
+	"block.sweetmagic.sticky_stuff_plant": "黏液花植株",
+	"block.sweetmagic.quartz_plant": "石英植株",
+	"block.sweetmagic.raspberry_plant": "树莓植株",
+	"block.sweetmagic.mfchanger": "魔力生成器",
+	"block.sweetmagic.mftable": "魔力充能台",
+	"block.sweetmagic.mftank": "魔力储罐",
+	"block.sweetmagic.mftank_adavance": "改良魔力储罐",
+	"block.sweetmagic.obmagia": "魔力书写台",
+	"block.sweetmagic.obmagia_top": "魔力书写台",
+	"block.sweetmagic.pedestal_creat": "魔力合成基座",
+	"block.sweetmagic.altar_creat": "魔力合成祭坛",
+	"block.sweetmagic.aether_hopper": "以太漏斗",
+	"block.sweetmagic.aether_repair": "以太修复台",
+	"block.sweetmagic.enchant_educe": "撰魔台",
+	"block.sweetmagic.magia_rewrite": "魔力转写台",
+	"block.sweetmagic.aether_reverse": "以太反转台",
+	"block.sweetmagic.magia_drawer": "魔力抽屉",
+	"block.sweetmagic.aether_lanp": "以太灯笼",
+	"block.sweetmagic.aether_crystal_ore": "以太水晶矿石",
+	"block.sweetmagic.deep_aether_crystal_ore": "深层以太水晶矿石",
+	"block.sweetmagic.alternative_block": "亚钢块",
+	"block.sweetmagic.aethercrystal_block": "以太水晶块",
+	"block.sweetmagic.divinecrystal_block": "神圣水晶块",
+	"block.sweetmagic.purecrystal_block": "纯净水晶块",
+	"block.sweetmagic.aethercrystal_block_alpha": "以太水晶块",
+	"block.sweetmagic.sugarglass": "糖玻璃",
+	"block.sweetmagic.shading_sugarglass": "深色糖玻璃",
+	"block.sweetmagic.frosted_glass": "磨砂玻璃",
+	"block.sweetmagic.frosted_glass_line": "划线磨砂玻璃",
+	"block.sweetmagic.prismglass": "棱镜玻璃",
+	"block.sweetmagic.shading_prismglass": "深色棱镜玻璃",
+	"block.sweetmagic.green4panel_glass": "十字镶框玻璃（绿）",
+	"block.sweetmagic.lightbrown4panel_glass": "十字镶框玻璃（浅棕）",
+	"block.sweetmagic.brown4panel_glass": "十字镶框玻璃（棕）",
+	"block.sweetmagic.darkbrown4panel_glass": "十字镶框玻璃（深棕）",
+	"block.sweetmagic.ami_glass": "嵌丝玻璃",
+	"block.sweetmagic.gorgeous_glass": "玻璃窗（黑）",
+	"block.sweetmagic.gorgeous_glass_w": "玻璃窗（白）",
+	"block.sweetmagic.magicbarrier": "魔力屏障玻璃",
+	"block.sweetmagic.sugarglass_pane": "糖玻璃板",
+	"block.sweetmagic.shading_sugarglass_pane": "深色糖玻璃板",
+	"block.sweetmagic.frosted_glass_pane": "磨砂玻璃板",
+	"block.sweetmagic.frosted_glass_line_pane": "划线磨砂玻璃板",
+	"block.sweetmagic.prismglass_pane": "棱镜玻璃板",
+	"block.sweetmagic.shading_prismglass_pane": "深色棱镜玻璃板",
+	"block.sweetmagic.green4panel_glass_pane": "十字镶框玻璃板（绿）",
+	"block.sweetmagic.lightbrown4panel_glass_pane": "十字镶框玻璃板（浅棕）",
+	"block.sweetmagic.brown4panel_glass_pane": "十字镶框玻璃板（棕）",
+	"block.sweetmagic.darkbrown4panel_glass_pane": "十字镶框玻璃板（深棕）",
+	"block.sweetmagic.ami_glass_pane": "嵌丝玻璃板",
```

```suggestion
	"block.sweetmagic.ami_glass_pane": "夹丝玻璃板",
```
[术语在线](https://www.termonline.cn/wordDetail?termName=%E5%A4%B9%E4%B8%9D%E7%8E%BB%E7%92%83&subject=0a4cb19826a911eea734b068e6519520&base=1)